deploy/initial-setup/setup.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+FILE=raspios_lite_armhf-2021-11-08/2021-10-30-raspios-bullseye-armhf-lite.zip
+FILENAME=$(basename $FILE)
+wget -nc https://downloads.raspberrypi.org/raspios_lite_armhf/images/$FILE
+unzip -n "$FILENAME"
+
+git clone --depth 1  --branch rpi-4.19.y https://github.com/Williangalvani/linux
```

Comment:
Can we use the official linux repository + branch and add the navigator file here ?
I have done it in https://github.com/bluerobotics/companion-docker/pull/705 as well, but I just saw the rework of the docker file on this PR now.

Reply:
It would be nice to have a readme with instructions as well.

Reply:
> Can we use the official linux repository + branch and add the navigator file here ? I have done it in #705 as well, but I just saw the rework of the docker file on this PR now.

yep, I think that is the way to go. I'll do it.